docker/build-tools/Dockerfile

```diff
@@ -74,10 +74,14 @@ FROM golang:1.12 as golang_context
 # Build a base os context
 FROM ubuntu:xenial as base_os_context
 
+# required for golang: ca-certificates, gcc, libc-dev,git
+# required for general build: make, wget, curl
```

Comment:
it would be helpful if we could standardize on either wget or curl in the build.  I'd prefer `curl`.